### PR TITLE
XP-261 move tests to tests dir

### PR DESCRIPTION
**Pull request by wilk10**
### References

https://xainag.atlassian.net/browse/XP-261

### Summary

Move tests to its own dir

### Are there any open tasks/blockers for the ticket?

None

---

### Reviewer checklist

*Reviewer agreement:*

* Reviewers assign themselves at the start of the review.
* Reviewers do **not** commit or merge the merge request.
* Reviewers have to check and mark items in the checklist.

**Merge request checklist**

- [x] Conforms to the merge request title naming `XP-XXX <a description in imperative form>`.
- [x] Each commit conforms to the naming convention `XP-XXX <a description in imperative form>`.
- [x] Linked the ticket in the merge request title or the references section.
- [x] Added an informative merge request summary.

**Code checklist**

- [x] Conforms to the branch naming `XP-XXX-<a_small_stub>`.
- [x] Passed scope checks.
- [x] Added or updated tests if needed.
- [x] Added or updated code documentation if needed.
- [x] Conforms to Google docstring style.
- [x] Conforms to XAIN structlog style.

### CODE REVIEW

File: tests/aggregate_test.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import numpy as np
 
-from . import aggregate
+from xain_fl.fl.coordinator.aggregate import federated_averaging
```
Comment (PanicButtonPressed) [inline tests/aggregate_test.py]: :thumbsup:

File: pytest.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [pytest]
-testpaths = benchmarks xain_fl
+testpaths = benchmarks xain_fl tests
```
Comment (PanicButtonPressed) [inline pytest.ini]: Do we want to separate in `unit` and `integration` tests already or do we have a dedicated ticket for it? If not, can you create one?

File: pytest.ini [hunk 3]
```diff
@@ -1,5 +1,5 @@
 [pytest]
-testpaths = benchmarks xain_fl
```
Comment (PanicButtonPressed) [inline pytest.ini]: Feel free to add a comment here to remove testpath `xain-fl`, once the benchmarks were moved out.

File: pytest.ini (hunk 2 above)
Comment (wilk10) [reply pytest.ini]: thanks, created: https://xainag.atlassian.net/browse/XP-352

File: pytest.ini (hunk 3 above)
Comment (wilk10) [reply pytest.ini]: makes sense! added

File: pytest.ini (hunk 2 above)
Comment (PanicButtonPressed) [reply pytest.ini]: Nice. Resolved.

File: pytest.ini (hunk 3 above)
Comment (PanicButtonPressed) [reply pytest.ini]: Thanks.

**Comment by wilk10** [comment]
closing, created a new PR against development: https://github.com/xainag/xain-fl/pull/197

**Pull request closed without merging**